### ISSUE TITLE
Weave Scope is not displaying all the containers running on my system

### ISSUE THREAD

**Issue by tomwilkie** [weaveworks/scope#210]
Submitter: Lokesh Dokara
Email: lokeshwithuever@gmail.com
Weave Scope is not displaying all the containers running on my system

**Comment by tomwilkie** [comment]
Hi Lokesh; thanks for the feedback!

Right now, Scope is only show containers with active tcp connections.  We're just doing the plumbing now to show all containers - see #171.

**Comment by tomwilkie** [comment]
I think @eldos-dl is real reporter.

Lokesh, does scope miss containers with connections, or does the above comment fit?

**Comment by eldos-dl** [comment]
I have an active connection to my database container which is not shown. Does that count?

**Comment by tomwilkie** [comment]
Yes that counts.

Did you build from source or pull from the dockerhub?  What version?

**Comment by eldos-dl** [comment]
I used the source git.io/scope-latest

**Comment by tomwilkie** [comment]
Thanks Lokesh; I think we've got a reproduction of the issue here.  Could you try a few things for me please?
- send us the output of `docker logs weavescope`
- let me know the output of `uname -a`
- try using the scope script here https://github.com/weaveworks/scope/releases/download/latest_release/scope

**Comment by eldos-dl** [comment]
```
$ uname -a
Linux ubuntu 3.11.0-19-generic #33~precise1-Ubuntu SMP Wed Mar 12 21:16:27 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux
$ docker images | grep scope
weaveworks/scope            latest              55bbbaaa7aff        17 hours ago        25.39 MB
weaveworks/scope            0.2.0               f1a559ec0874        9 days ago          25.99 MB
```
Using script from [https://github.com/weaveworks/scope/releases/download/latest_release/scope](https://github.com/weaveworks/scope/releases/download/latest_release/scope)
```
$ docker logs weavescope
2015/06/11 09:42:04 app starting, version 0.2.0
2015/06/11 09:42:04 probe starting, version 0.2.0
2015/06/11 09:42:04 listening on :4040
2015/06/11 09:42:04 dialing 127.0.0.1:4030 (backoff 2s)
2015/06/11 09:42:04 connected to 127.0.0.1:4030
2015/06/11 09:42:04 connection initiated: 127.0.0.1:46380
2015/06/11 09:42:04 lookup scope.weave.local: lookup scope.weave.local: no such host
2015/06/11 09:42:11 listening on :4030
2015/06/11 09:43:04 lookup scope.weave.local: lookup scope.weave.local: no such host
```
Using script from [https://git.io/scope-latest](https://ggit.io/scope-latest)
```
$ docker logs weavescope
2015/06/11 09:47:52 probe version 9c6ed7b
2015/06/11 09:47:52 app starting, version 9c6ed7b
2015/06/11 09:47:52 listening on :4040
2015/06/11 09:47:52 dialing 127.0.0.1:4030 (backoff 2s)
2015/06/11 09:47:52 connected to 127.0.0.1:4030
2015/06/11 09:47:52 connection initiated: 127.0.0.1:46514
2015/06/11 09:47:52 listening on :4030
```

**Comment by tomwilkie** [comment]
Thanks Lokesh; when using the 0.2.0 release, did you see your containers?

**Comment by eldos-dl** [comment]
Nope, I couldn't see

**Comment by tomwilkie** [comment]
Hi Lokesh; we released 0.3 on Monday, could you give it a try on that release please?  It should show all containers regardless of whether they are making any network connections.

We also might have uncovered the reason why we're not seeing your containers making network calls (#245) and we're working on a fix.

**Comment by tomwilkie** [comment]
We believe this to be fixed now.
